<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ETCView</class>
 <widget class="QWidget" name="ETCView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>694</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupChannel">
       <property name="title">
        <string>Channel Selection</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QComboBox" name="comboChannel">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Disp(SN/Vol)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="0">
            <widget class="QPushButton" name="btnDispSet">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QSpinBox" name="spinDispStripNumber">
             <property name="minimum">
              <number>-2147483647</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="spinDispVolumn">
             <property name="minimum">
              <number>-2147483647</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Asp(SN/Time)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QSpinBox" name="spinAspStrip">
             <property name="minimum">
              <number>-2147483647</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="spinAspTime">
             <property name="minimum">
              <number>-2147483647</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="btnAspRun">
             <property name="text">
              <string>Run</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Prime(Sec)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QSpinBox" name="spinPrimeTime">
             <property name="minimum">
              <number>-2147483647</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btnPrimeSet">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Channel :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <widget class="QGroupBox" name="gbFanTest">
       <property name="title">
        <string>Fan Test</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="1">
         <widget class="QPushButton" name="btnFanTestOFF">
          <property name="text">
           <string>OFF</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnFanTestON">
          <property name="text">
           <string>ON</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="gbLedTest">
       <property name="title">
        <string>Status LED</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="1">
         <widget class="QPushButton" name="btnStatusLEDOFF">
          <property name="text">
           <string>OFF</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnStatusLEDON">
          <property name="text">
           <string>ON</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="groupBox_10">
       <property name="title">
        <string>Incubation(Sec)</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0" colspan="2">
         <widget class="QSpinBox" name="spinIncubationTime">
          <property name="minimum">
           <number>-2147483647</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="btnIncubation">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="gbAirTempControl">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Air Temperature</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinAirTemperatureVolt">
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinAirTemperatureResult">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinAirTemperatureB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>3435.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Volt</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Result</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="btnAirTemperature">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="gbTrayTempControl">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Tray Temperature</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_17">
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Tray Temp(C)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Volt</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinTrayTemperatureB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>3435.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinTrayTemperatureVolt">
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinTrayTemperatureValue">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>StripTemp</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinStripTemp">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>23</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="btnTrayTemperature">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>CpValue</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="spinTrayCompensationValue">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>39.180999999999997</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item row="2" column="0">
      <widget class="QGroupBox" name="gbTempCtrl">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>HeatingPad Control</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="1">
         <widget class="QPushButton" name="btnHeatingPadOFF">
          <property name="text">
           <string>OFF</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnHeatingPadON">
          <property name="text">
           <string>ON</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="gbTempCh1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Heating Pad Temp Ch1</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0" colspan="2">
         <widget class="QSpinBox" name="spinHeatingPadTempCh1">
          <property name="minimum">
           <number>-2147483647</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnHeatingPadTempCh1Read">
          <property name="text">
           <string>Read</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnHeatingPadTempCh1Write">
          <property name="text">
           <string>Write</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="gbTempCh2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Heating Pad Temp Ch2</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0" colspan="2">
         <widget class="QSpinBox" name="spinHeatingPadTempCh2">
          <property name="minimum">
           <number>-2147483647</number>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnHeatingPadTempCh2Read">
          <property name="text">
           <string>Read</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnHeatingPadTempCh2Write">
          <property name="text">
           <string>Write</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="gbAutoClean">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Autoclean</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_16">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnAutoclean">
          <property name="text">
           <string>AutoClean</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
